plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -364,6 +365,7 @@ class FeatureDevController(
 
         messenger.sendAnswer(tabId = tabId, messageType = FeatureDevMessageType.Answer, message = message("amazonqFeatureDev.chat_message.ask_for_new_task"))
```

Comment:
Can you extract this message out of newTask function? Otherwise, it would display when NoChangeRequiredException happens as well.

Reply:
Sure, but I don't see it you removing on the VS Code part. Are we extracting there too?

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/5723  L135 and L776 in `packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts`

Reply:
My question was more about the placeholder. Synced offline.